README.md

```diff
@@ -1,9 +1,8 @@
-## Custom App Surfaces SDK
+## Custom UI extensions SDK
```

Comment:
Let's change it to "App extensions SDK" for the future renaming the package to `app-extensions-sdk`